tests/integration/jinad/test_index_query_with_shards/test_integration.sh

```diff
@@ -56,7 +56,7 @@ docker-compose -f tests/integration/jinad/test_index_query/docker-compose.yml --
 
 EXPECTED_TEXT='"text:hey, dude"'
 
-if [ "$EXPECTED_TEXT" = "TEXT_MATCHED" ]; then
+if [ "$EXPECTED_TEXT" = "$TEXT_MATCHED" ]; then
```

Comment:
It is always better to use bash instead of default sh, and `if [[ "$a" == "$b" ]]` instead of `[ "$a" = "$b" ]`